### DISCUSSION TITLE
Migration Confusion

### DISCUSSION THREAD

**Discussion by innovabin-dev** [TriliumNext/Trilium#4525] [Q&A]
Hello. I wasn't exactly sure where to put this, so I'm asking it as a Q&A. I've searched high and low on stack exchange, et cetera, but don't see any reference to this particular scenario.

**PROBELM**
- _I have an old v197 database which I'm trying to migrate._
- _Starting with a newer-but-not-newest server instance, I tried "stepping" my way up through migrations._
- _Newer server instance uses db version 212, which migrates nicely...but in the process it continues to try and migrate to versions BEYOND 212, until it fails when starting the upgrade to 216._

How is this possible? Is there a cache somewhere that remembers versions beyond the currently installed server instance?
I've tried three or four DIFFERENT Trilium releases, and encounter the same effect. All fail at 216.

Thanks,
Angus

**Comment by Nriver** [comment]
Upgrade to 0.60.4 first, then the latest.
